utils/generateUserMappings.js

```diff
@@ -0,0 +1,41 @@
+const users = require('../data/users.json')
+const fs = require('fs');
+
+users.forEach(user => {
+  const filename = `CommunityAPI_GetUser_${user.username}.json`
+  const body = {
+    "name": `community_api_get_user_${user.username}`,
+    "request": {
+      "url": `/secure/staff/username/${user.username}`,
+      "method": "GET"
+    },
+    "response": {
+      "status": 200,
+      "jsonBody": {
+        "username": user.username,
+        "email": user.email,
+        "telephoneNumber": "01512112121",
+        "staffCode": "SH00007",
+        "staffIdentifier": 17,
+        "staff": {
+            "forenames": user.name,
+            "surname": ""
+        },
```

Comment:
I think that rather soon we will need this request to the Community API to return information about the user's team memberships (`"teams": [...]`) so that we can test whether the CRN of the application is within the caseload of that team. But that can be added later once we've implemented the auth control on who an "applicant" can see..

Reply:
Yeah, that's what I was thinking, but yeah, we can iterate as we go